### PR TITLE
use Altinn bearer httpClient extensions

### PR DESCRIPTION
**Pull request by simen-rekkedal**
## Description

Change to use Altinn Authentication httpClient Extension methods with bearer token

## Related Issue(s)
- #{issue number}

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI.Integration/SystemUsers/SystemUserClient.cs [hunk 1]
```diff
@@ -40,21 +62,21 @@ public SystemUserClient(HttpClient httpClient)
         return null;
     }
 
-    public async Task<SystemUserReal?> PostNewSystemUserReal(SystemUserDescriptor newSystemUserDescriptor, CancellationToken cancellation = default)
+    public async Task<SystemUserReal?> PostNewSystemUserReal(
```
Comment (acn-dgopa) [inline bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI.Integration/SystemUsers/SystemUserClient.cs:65]: Waht does Real mean here

File: bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI.Integration/SystemUsers/SystemUserClient.cs [hunk 2]
```diff
@@ -40,21 +62,21 @@ public SystemUserClient(HttpClient httpClient)
...
+        SystemUserDescriptor newSystemUserDescriptor, 
+        CancellationToken cancellation = default)
     {
+        string token = JwtTokenUtil.GetTokenFromContext(_httpContextAccessor.HttpContext!, _platformSettings.JwtCookieName!)!;
+        string endpointUrl = $"authentication/api/v1/systemuser";
+        //var accessToken = await _accessTokenProvider.GetAccessToken();
```
Comment (acn-dgopa) [inline bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI.Integration/SystemUsers/SystemUserClient.cs:71]: dont we need an accesstoken here

File: bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI/Controllers/SystemUserController.cs [hunk 3]
```diff
@@ -111,7 +111,7 @@ public async Task<ActionResult> UploadCertificate([FromForm] IFormFile file, [Fr
         return Ok();
     }
     
-    [Authorize]
+    //[Authorize]
```
Comment (acn-dgopa) [inline bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI/Controllers/SystemUserController.cs:114]: Comment needs to be removed

File: bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI.Integration/SystemUsers/SystemUserClient.cs (hunk 1 above)
Comment (simen-rekkedal) [reply bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI.Integration/SystemUsers/SystemUserClient.cs:65]: In the BFF we have descriptions of new SystemUsers coming in from the Frontend with the SystemUserDescriptor Model. Returning is the SystemUserReal from the Authentication component. So Real is used as a suffix to mean that it exists on the db in the Authentication component.

File: bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI/Controllers/SystemUserController.cs (hunk 3 above)
Comment (simen-rekkedal) [reply bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI/Controllers/SystemUserController.cs:114]: ok

File: bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI.Integration/SystemUsers/SystemUserClient.cs (hunk 2 above)
Comment (simen-rekkedal) [reply bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI.Integration/SystemUsers/SystemUserClient.cs]: ok fixed

**Review by acn-dgopa — APPROVED** [review]

**Pull request merged**